cqproto/grpc.go

```diff
@@ -338,6 +339,7 @@ func diagnosticsToProto(in diag.Diagnostics) []*internal.Diagnostic {
 					Summary:  r.Description().Summary,
 					Detail:   r.Description().Detail,
 					Resource: r.Description().Resource,
+					//ResourceId: p.Description().ResourceID,
```

Comment:
?

Reply:
I thought "let's not put IDs in redacted diagnostics in the first place, so they won't get passed down even if they're set"

Reply:
But I removed this special treatment, thinking redacted diags might still have the resource IDs or some form of them if the provider developer wants.